app/modules/nep9/Nep9QrGenerator.js

```diff
@@ -0,0 +1,127 @@
+// @flow
+/* eslint-disable */
+import QRCode from 'qrcode/lib/browser'
+import nep9 from './nep9'
+import tokenList from '../../core/tokenList.json'
+import { imageMap } from '../../assets/nep5/raw-svg-export'
+
+const TOKENS = Object.keys(tokenList)
+  .map(key => tokenList[key])
+  .reduce((accum, token) => {
+    accum[token.networks[1].hash] = token
```

Comment:
this is prob ok if the list is hardcoded but I still prefer lodash get

Reply:
Because its hardcoded and cannot change I am going to leave this as is for now